ESP_SHT3x/ESP_SHT3x.ino

```diff
@@ -32,14 +106,50 @@ void printShtMeasurments(float temp, float hum) {
     Serial.println("Failed to read humidity");
   }
 }
- 
-void loop() {
-  shtInit();
+
+// Sets relay pins to high or low depending on measurements
+void relayPower(int pin,float current, float limit) {
+  if(current < limit) {
+    digitalWrite(pin, LOW);
+  } else {
+    digitalWrite(pin, HIGH);
+  }
+}
+
+
+void shtAlgorithm() {
+  // Reads temperature and humidity from sht31 and stores it to appropriate variables
   float t = sht31.readTemperature();
   float h = sht31.readHumidity();
-
+  
+  // Displays current readings
   printShtMeasurments(t, h);
   
+  // Sets relay relevent pins to appropriate voltage levels
+  relayPower(humidPin, h, humidLimit);
+  relayPower(tempPin, t, tempLimit);
+}
+
+void DS18B20Algorithm() {
+  sensors.requestTemperatures(); 
+  float temperatureC = sensors.getTempCByIndex(0);
+
+  relayPower(tempPin, temperatureC, tempLimit);
+  
+  Serial.print(temperatureC);
+  Serial.println("ºC");
+}
+ 
+void loop() {
+  if(sht31.begin(0x44)) {
+    shtAlgorithm();
+  }else {
```

Comment:
spacing

Reply:
also, we'll want to make this an `else if` and include a check to ensure the backup thermometer is accessible.

In the case that neither the SHT31 or the DS18B20 are working, we should fall back to a timer based system (once an hour, on for one minute)